index.js

```diff
@@ -0,0 +1,39 @@
+/** *** *** **/
+/** CONFIG  **/
+/** *** *** **/
+var app = require('express')();
+var fs = require('fs');
+var pem = fs.readFileSync('./cert/wildcard.pem');
+var credentials = {key: pem, cert: pem};
+var server = require('https').Server(credentials, app);
+var io = require('socket.io')(server);
+var proxyCaps = require('./splitter');
+var config = require('./config');
+var bodyParser = require('body-parser');
+config.seleniumClients = proxyCaps;
+
+var _ = require('lodash');
+
+var mng = require('./mng')(config);

```

Comment:
lol, when i m prototyping on my knee i dont care. When I have time I try to avoid ;]